requirements.txt

```diff
@@ -1,45 +1,42 @@
 authutils==3.1.1
-datamodelutils==0.4.7
+datamodelutils==1.0.0
 defusedxml==0.5.0
-dictionaryutils==2.0.7
-gen3dictionary==2.0.0
+dictionaryutils==3.0.0
+gen3dictionary==2.0.1
 SurvivalPy==1.0.2
 simplejson==3.8.1
 stopit==1.1.1
-functools32==3.2.3-2
-lxml==3.4.1
 python-Levenshtein==0.12.0
 python-keystoneclient==1.6.0
 fuzzywuzzy==0.6.1
 addict==0.2.7
 Flask==1.1.1
 Flask-Cors==1.9.0
 gen3authz==0.2.1
-Jinja2==2.7.3
+Jinja2==2.10.1
+lxml==4.4.1
 MarkupSafe==0.23
 PyYAML==5.1
 Werkzeug==0.16.0
 boto==2.36.0
 elasticsearch==1.2.0
 itsdangerous==0.24
 requests== 2.22.0
-six==1.10.0
-urllib3==1.24.1
-wsgiref==0.1.2
+six==1.12.0
```

Comment:
we don't, deleting

Reply:
removing it caused travis to fail so I added it back. It's weird because psqlgraph does use six but it also specifies in [setup.py](https://github.com/uc-cdis/psqlgraph/blob/develop/setup.py#L30) so not sure why it's not being properly installed on travis.

Reply:
ugh, ok